kohese-portal/client/src/components/type-editor/view-model-editor/view-model-editor.component.ts

```diff
@@ -290,7 +293,7 @@ export class ViewModelEditorComponent {
       return !!input;
     });
     if (name) {
-      let id: string = (<any> Uuid).default();
+      let id: string = UuidV1;
```

Comment:
It looks like this will assign the contents of the Uuidv1 function to id.  Looking at the stored tableDefinitions, that looks like what happened...

Reply:
Did this mess up the stored tableDefinitions?